### ISSUE TITLE
Display of nicks in channel corrupted, due to fault with links?

### ISSUE THREAD

**Issue by akempgen** [colloquy/colloquy#618]
*Issue migrated from trac ticket [#337](http://colloquy.info/project/ticket/337)*

**type:** Defect | **component:** Colloquy (Mac) | **severity:** Blocker | **priority:** Normal | **version:** 2.0 (Mac) | **status:** closed | **resolution:** Invalid | **keywords:** links nicks display invisible hover
___
#### 2005-10-03 17:54:46: irbdavid@… created the issue

Having a lot of trouble reproducing this 'on demand' but here goes: [[BR]]
After a few minutes use, possibly after a link is given in a channel, hovering the mouse over the nicks of people speaking in the channel window will cause them to disappear, flushing the text (that they wrote) far right.  Hovering the mouse over links written in the channel also causes them to disappear.  [[BR]]
This occurs using the 'Standard' style, but only seems to affect individual channels (eg - if it occurs in one channel, it is persistent in that channel, but another channel in a separate window or tab will be unaffected. It also persists across switching tabs - this doesn't 'fix' it).  After a link / links disappear in this way, new messages in the channel will restore them, though hovering over links will still have the same effect.[[BR]][[BR]]
Sorry can't work out how to reliably reproduce this one.[[BR]]
-irbdavid
___
#### 2005-10-03 19:36:40: @kballard changed status from *new* to *closed*

___
#### 2005-10-03 19:36:40: @kballard changed resolution from ** to *invalid*

___
#### 2005-10-03 19:36:40: @kballard commented

This is a WebKit bug (nicks are affected because they are links). To fix it type `/reload style` and it will re-display the chat output with all the links restored.

So basically, it's not our bug, we don't know why it happens or how to fix it. But we have a workaround.
